src/sql/SqlResult.ts

```diff
@@ -98,39 +98,37 @@ export interface SqlResult extends AsyncIterable<SqlRowType> {
      * Returns row metadata of the result.
      * @returns SQL row metadata if rows exist in the result; otherwise null.
      */
-    getRowMetadata(): Promise<SqlRowMetadata | null>;
+    getRowMetadata(): SqlRowMetadata | null;
```

Comment:
Not related to here but, above, we should update `### Usage for update count` section and remove await